Cargo.toml

```diff
@@ -1,22 +1,6 @@
-[package]
-name = "portability"
-version = "0.1.0"
-authors = ["Dzmitry Malyshau <kvark@mozilla.com>"]
-
-[lib]
-name = "portability"
-crate-type = ["staticlib"]
-
-[dependencies]
-winit = "0.7"
-
-[dependencies.gfx-hal]
-#path = "../gfx/src/hal"
-git = "https://github.com/kvark/gfx-rs"
-branch = "portable"
-
-[dependencies.gfx-backend-vulkan]
-#path = "../gfx/src/backend/vulkan"
-git = "https://github.com/kvark/gfx-rs"
-branch = "portable"
-features = ["portable"]
+[workspace]
+members = [
+    "libportability",
```

Comment:
I'm not comfortable with "libportability-XXX" name. Is there any other lib that starts with "lib"?

Reply:
Adhering to the layout of the [`rust-lang/rust`](https://github.com/rust-lang/rust/tree/master/src) repo, but I don't mind other names. suggestions?

Reply:
I'm a bit confused by the fact the repo itself here is about the portability library, so having `libportability-x` inside it is redundant. This is unlike Rust repo.
However, I suppose we can end up with some binary sub-modules later on, so this is future-proof.